### ISSUE TITLE
Define a static list of main categories

### ISSUE THREAD

**Issue by erikaheidi** [dev-human/dev-human#15]
Define a static list to feature the main entry categories. This way we have some pre-defined "sections", for getting things more organized since when creating a new post you can now use whatever you want for categories - and they go straight to the main "category" menu.

**Comment by ramondelafuente** [comment]
Good point. The current list of categories has been maintained but it could be nice to fix it somewhat. And add a validation step to the build process.
